src/main/java/business_objects/Note.java

```diff
@@ -5,6 +5,14 @@
     private String name;
     private NoteType type;
     private Color color = Color.YELLOW;
```

Comment:
I think yes, we do. As by default we have yellow note. So in case we don't define the color for Note object, we have yellow color by default. Probably I am missing smth, then please, go ahead and share your point of view with me :)